README.md

```diff
@@ -92,6 +92,45 @@ example: `add_filter( 'pb_integrations_multidomain_email', function( $email, $ui
 Because this plugin uses the fabulous [onelogin/php-saml](https://github.com/onelogin/php-saml/)
 toolkit, [many other configuration variables can be tweaked](https://github.com/onelogin/php-saml/#settings).
 
+## Using a test IdP SAML app
+If you want to test the plugin with a test IdP, you can use [Auth0](https://auth0.com/).
+Here are the steps to create a test IdP application:
+1. Create an Auth0 account (or ask the Operation Team for a test account).
+2. Create a new application in the Auth0 dashboard.  
+3. Select "Regular Web Applications" as the application type.
+4. Go to the "Settings" tab of your application.
+5. Scroll down to the "Advanced Settings" section and click on "Ednpoints" tab.
+6. Copy the "SAML Metadata URL" and paste it Pressbooks SAML2 settings page, in the "IdP metadata URL" field and save the changes.
+7. In Auth0, go to the "Addons" tab of your application, and click on "SAML2 Web App".
+8. In the Settings tab, under the Application Callback URL, add: `https://<NETWORK_DOMAIN>/wp/wp-login.php?action=pb_shibboleth_acs`
+9. In the Settings JSON, add the following:
+  ```json
+	{
+		"audience": "https://<NETWORK_DOMAIN>/wp/shibboleth",
```

Comment:
To me this URL had to be configured as `https://<NETWORK_DOMAIN>/shibboleth`. If I add the `/wp` it fails. Might be a configuration thing though 🤔.